rocAL/source/pipeline/node.cpp

```diff
@@ -38,3 +38,18 @@ void Node::create(std::shared_ptr<Graph> graph) {
 void Node::update_parameters() {
     update_node();
 }
+
+void Node::add_next(const std::shared_ptr<Node> &node) {
+    
```

Comment:
Please remove the empty blank line at L43

Reply:
Done